kerberos.json

```diff
@@ -0,0 +1,126 @@
+{
+  "services": [
+    {
+      "name": "CDAP",
+      "identities": [
+        {
+          "name": "/spnego"
+        },
+        {
+          "name": "/hdfs"
+        },
+        {
+          "name": "cdap",
+          "principal": {
+            "value": "cdap@${realm}",
+            "type" : "user",
+            "configuration": "cdap-env/cdap_principal_name",
+            "local_username": "cdap"
+          },
+          "keytab": {
+            "file": "${keytab_dir}/cdap.headless.keytab",
```

Comment:
guessing this is just a label or something?  just pointing it out to doublecheck, as the others use cdap.master.kerberos.keytab.